### PR TITLE
Our new program: Pegasus

### PR DESCRIPTION
**Pull request by mawlm5007**

### CODE REVIEW

**Comment by jonhuo11** [comment]
Lgtm!

**Comment by ArnavPal** [comment]
LGTM! Great idea!

**Comment by jonhuo11** [comment]
@Jeromephilip Waiting on your approval for Q/A

**Comment by jonhuo11** [comment]
nit: Remove lines `0` to `1000`

File: README.md
Comment (jonhuo11) [inline README.md]: Lets make similarities to Bash, Golang, and Rust as well and deploy to Kubernetes over Apache Cloud

**Review by jonhuo11 — APPROVED** [review]
Looks great, one small change before Q/A

**Pull request closed without merging**
